### PR TITLE
Changes in WebProperties usage

### PR DESCRIPTION
**Pull request by fviale**

### CODE REVIEW

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/shared/Config.java [hunk 1]
```diff
@@ -110,21 +106,11 @@ public void reload() {
     }
 
     public boolean isHttpsAllowAnyCertificate() {
-        return getBooleanValue(HTTPS_ALLOW_ANY_CERTIFICATE, false);
+        return WebProperties.WEB_HTTPS_ALLOW_ANY_CERTIFICATE.getValueAsBoolean();
```
Comment (marcocast) [inline common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/shared/Config.java]: it doesn't return false by default?

Comment (fviale) [inline common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/shared/Config.java]: yes by default it returns false

**Comment by fviale** [comment]
Retest this please

**Comment by fviale** [comment]
Retest this please

**Pull request merged**